src/TurbulenceClosures/turbulence_closure_implementations/scale_invariant_smagorinsky.jl

```diff
@@ -76,14 +76,14 @@ function compute_diffusivities!(diffusivity_fields, closure::ScaleInvariantSmago
     velocities = model.velocities
     tracers = model.tracers
 
-    LM_op = KernelFunctionOperation{Center, Center, Center}(LᵢⱼMᵢⱼ_ccc, grid, velocities...)
-    MM_op = KernelFunctionOperation{Center, Center, Center}(MᵢⱼMᵢⱼ_ccc, grid, velocities...)
+    LM_op = Average(KernelFunctionOperation{Center, Center, Center}(LᵢⱼMᵢⱼ_ccc, grid, model.velocities...))
+    MM_op = Average(KernelFunctionOperation{Center, Center, Center}(MᵢⱼMᵢⱼ_ccc, grid, model.velocities...))
 
-    LM_avg = Field(Average(LM_op, dims=closure.averaging.dims))
-    MM_avg = Field(Average(MM_op, dims=closure.averaging.dims))
+    mean!(diffusivity_fields.LM_avg, LM_op.operand)
+    mean!(diffusivity_fields.MM_avg, MM_op.operand)
```

Comment:
Note this only works on uniform grids, but hopefully that's good enough for preliminary testing